EIC-EA/EIC-community-D8
# Feature/eicnet 1532 group content event detailed page

### How to test

- Go to storybook bundles/group/group event detailed
- check if it's ok with the [figma](https://www.figma.com/file/WZH3qhNynnmm0H6FKmD3hi/EASME_v1?node-id=9118%3A40870)
